pkg/util/mount/mount_linux.go

```diff
@@ -503,8 +514,14 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 			return err
 		}
 		if existingFormat == "" {
+			if readOnly {
+				// Don't attempt to format if mounting as readonly, return an error to reflect this.
+				glog.Errorf("Failed to mount unformatted volume %q as read only", source)
```

Comment:
Is this necessary? Is the error returned one line below logged somewhere?

Reply:
I read the code again and test in my local environment, found that the error will be logged by this [line]()(https://github.com/kubernetes/kubernetes/blob/b9361192b806df8a9f4108d97f3fe47ac61d1ccf/pkg/volume/util/nestedpendingoperations/nestedpendingoperations.go#L267), as this error will be returned by each volume plugins' `MountDevice`, and handled by `nestedPendingOperations`. So I remove the log here, PTAL.